packages/oui-navbar/src/navbar.html

```diff
@@ -0,0 +1,151 @@
+<!-- Toggler -->
+<button class="oui-navbar-toggler" type="button"
```

Comment:
Not a big fan of over-cutting template, but it could be done for the notifications menu.
The toggler and the user menu is already covered by `oui-navbar-menu` :)